README.md

```
@@ -89,49 +87,39 @@ To run this file, make sure you have [Docker installed](https://docs.docker.com/
+Currently, we only allow authentication via GitHub. To enable this, you need to have a `GITHUB_ID` and `GITHUB_SECRET` value.
 
-Setup your GitHub ID & Secret on GitHub:
+Set up your GitHub ID & Secret on GitHub:
 
-- [Click here](https://github.com/settings/applications/new) to setup New OAuth App and fill in the details as shown below.
+- [Click here](https://github.com/settings/applications/new) to set up a New OAuth App and fill in the details as shown below.
 
-For development, make sure you setup this with a **Homepage URL** of
+For development, make sure you set this up with a **Homepage URL** of
 
 ```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)